_docs/rfc/dynamo_table_config.md

```diff
@@ -0,0 +1,47 @@
+# Allow further customization of Terraform Lock table for S3 Remote State
+
+**STATUS**: In proposal
+
+## Background
+
+When Terragrunt creates a DynamoDB table, it sets the throughput to the default,
+`PROVISIONED`. Provisioned is good for stable throughput, but Terraform largely
+works with variable workloads, often going unused.
+
+AWS also offers a `PAY_PER_REQUEST` billing mode that charges based on the
+number of requests to the lock table. The pricing is dirt cheap, and it will
+further reduce the costs associated with using Terragrunt to manage Terraform's
+base requirements.
+
+Terraform has solved this problem: It involves creating and managing the
+resource manually. But, if Terragrunt is going to be managing the table, it
+should allow for additional configuration to be layered on top of it via
+Terragrunt.
```

Comment:
Makes a lot of sense! This actually has a lot of parallels with the challenge of setting up the S3 bucket (see https://github.com/gruntwork-io/terragrunt/issues/924).